mod/groups/start.php

```diff
@@ -81,7 +81,10 @@ function groups_init() {
 
 	//extend some views
 	elgg_extend_view('elgg.css', 'groups/css');
-	elgg_extend_view('elgg.js', 'groups/js');
+	if (_elgg_view_may_be_altered('groups/js', __DIR__ . '/views/default/groups/js.php')) {
```

Comment:
I wasn't clear. I was suggesting we alter the function to accept an absolute path (which it doesn't now).